psi4/src/psi4/libpsio/filemanager.cc

```diff
@@ -42,17 +42,7 @@ namespace psi{
 PSIOManager::PSIOManager()
 {
     pid_ = psio_getpid();
-
-    // set the default to /tmp unless one of the
-    // TMP environment variables is set
-    if(std::getenv("TMPDIR"))
-        set_default_path(std::getenv("TMPDIR"));
-    else if(std::getenv("TEMP"))
-        set_default_path(std::getenv("TEMP"));
-    else if(std::getenv("TMP"))
-        set_default_path(std::getenv("TMP"));
-    else
-        set_default_path("/tmp");
+    set_default_path("/tmp");
```

Comment:
Just to be absolutely sure, you checked that `psidatadir` and `PSIO` still play nicely together? It might be good to engineer a unittest for that at some point.

Reply:
I checked with all the cleaning commented out:
* plain `psi4`
* `psi4 -s scr`
* `PSI_SCRATCH=scr2 psi4 -s scr`
* `PSI_SCRATCH=scr2 psi4`
* `python input.py`
* `PSI_SCRATCH=scr2 python input.py`

and they all _seem_ to work as advertised. But yes, I'm nervous, too.

Actually, now that I read again, what do `psidatadir` and `PSIO` have to do with each other?

I did also test `psi4 -l alt_psidatadir` and it at least prints in the right place and runs.